CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 * Fixes an issue with having clause when used with complex joins
 
+### citus v8.0.3 (January 9, 2019) ###
```

Comment:
Can we keep the versions in time-sorted? I mean putting that version at the top.

Reply:
Makes sense. Ordering by the version numbers looked like the better option to me. I guess this is not my decision to make though.